nRFMeshProvision/ProxyFilter.swift

```diff
@@ -84,19 +104,29 @@ public extension ProxyFilterDelegate {
 /// An enumeration for different initial configurations of the Proxy Filter.
 public enum ProxyFilterSetup {
     /// In automatic Proxy Filter setup the filter will be set to
-    /// ``ProxyFilerType/inclusionList`` with Unicast Addresses of all
+    /// ``ProxyFilerType/acceptList`` with Unicast Addresses of all
     /// local Elements, all Group Addresses with at least one local Model
     /// subscribed and the All Nodes (0xFFFF) address.
     ///
     /// This is the default configuration.
     case automatic
     
     /// The Proxy Filter on each connected Proxy Node will be set to
-    /// ``ProxyFilerType/inclusionList`` with given set of addresses.
+    /// ``ProxyFilerType/acceptList`` with given set of addresses.
+    case acceptList(addresses: Set<Address>)
+    
+    /// The Proxy Filter on each connected Proxy Node will be set to
+    /// ``ProxyFilerType/rejectList`` with given set of addresses.
+    case rejectnList(addresses: Set<Address>)
```

Comment:
This looks like a typo? 
```diff
-rejectnList
+rejectList
```

There are some more of the same typo in this file...